themes/example/lms/templates/courseware/course_about.html

```diff
@@ -178,12 +178,25 @@
 
 <section class="course-info" xmlns="http://www.w3.org/1999/html">
   <header class="hero-main grid-manual hero-video">
+
+    <%doc>
+      Field name for background hero image: hero_image_asset_path
+
+      TODO: Hero images is applying as background in scss, in hero-video class, Here are thing we can do:
+      1. Use reletive path and keep this in SCSS (_overrides.scss:922)
+      2. Move background property to inline style
+      3. Update structure, move that in HTML:
+        - use image in image tag, with very low z-index
+        - float the other elements above that image
+        - or adjust them by giving the margin
+    </%doc>
```

Comment:
@malikshahzad228 Let's go with option 3 to bring this more inline with the home page.